sources/accessible-name-and-description.ts

```diff
@@ -378,13 +386,12 @@ export function computeTextAlternative(
 			// TODO: Unclear why display affects delimiter
 			// see https://github.com/w3c/accname/issues/3
 			const display = isElement(child)
-				? getComputedStyle(child).getPropertyValue("display")
+				? (context.style || getComputedStyle(child)).getPropertyValue("display")
```

Comment:
How do we know that `context.style` is the style of `child`?

Reply:
you're right, this is incorrect. doing it with a caching `getComputedStyle` function was the right way to do it